### PR TITLE
Add Cookie Store and Partitioned Cookies features

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adds two features related to cookie storage: Cookie Store and Partitioned Cookies.

### CODE REVIEW

File: features/partitioned-cookies.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+spec: https://wicg.github.io/cookie-store/
+compat_features:
+  - api.CookieChangeEvent.changed.partitioned_property
+  - api.CookieChangeEvent.deleted.partitioned_property
+  - api.CookieStore.delete.partitioned_option
+  - api.CookieStore.get.partitioned_return_property
+  - api.CookieStore.getAll.partitioned_return_property
+  - api.CookieStore.set.partitioned_option
+  - api.ExtendableCookieChangeEvent.changed.partitioned_property
+  - api.ExtendableCookieChangeEvent.deleted.partitioned_property
```
Comment (ddbeck) [inline features/partitioned-cookies.yml:12]: This looks incomplete. CHIPS is a bigger feature than the cookie store API. This is missing at least an HTTP header, but I don't know enough about it to say what else might be missing.

File: features/cookie-store.yml [hunk 2]
```diff
@@ -1,35 +1,27 @@
-draft_date: 2024-09-09
-name: Cookie Store API
-description: TODO
+name: Cookie store
+description: To write...
```
Comment (ddbeck) [inline features/cookie-store.yml:2]: Do you want to take a crack at writing these descriptions? Or am I jumping the gun with a review and this was supposed to be a draft PR?

**Review by ddbeck — CHANGES_REQUESTED** [review]

File: features/cookie-store.yml (hunk 2 above)
Comment (queengooborg) [reply features/cookie-store.yml:2]: I'm not good at writing descriptions, so I kind of left this as a request for assistance writing one!

File: features/partitioned-cookies.yml (hunk 1 above)
Comment (ddbeck) [reply features/partitioned-cookies.yml:12]: Want to come back to this PR and feature, @queengooborg?

Comment (Elchi3) [reply features/partitioned-cookies.yml:12]: I added http.headers.Set-Cookie.Partitioned

**Review by autonome — DISMISSED** [review]

File: features/partitioned-cookies.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: Partitioned cookies
+description: Cookies Having Independent Partitioned State (CHIPS, also known as Partitioned cookies) allows developers to opt a cookie into partitioned storage, with a separate cookie jar per top-level site.
```
[the remark this answers is not in the document]
Comment (ddbeck) [reply features/partitioned-cookies.yml:2]: I don't think we need the "also known as" when it's in the name of the feature, but it's not written down anywhere. That said, I'd probably write it like this:

```suggestion
description: Partitioned cookies allow you to opt a cookie into partitioned storage, with a separate cookie jar per top-level site. Also known as Cookies Having Independent Partitioned State or CHIPS.
```

**Review by ddbeck — APPROVED** [review]
I think you're free to merge this, after tending to the description discussion.

**Pull request merged**